File: backend/products/src/main/java/com/spring/products/entity/ServerDetails.java
package com.spring.products.entity;

import com.spring.products.enumeration.Status;
import lombok.*;

import javax.persistence.Entity;

@Entity
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class ServerDetails extends BaseEntity {
    private String physicalServer;
    private String serverName;
    private String ipAddress;
    private String userName;
    private String password;
    private Status statusServer = Status.ACTIVE;
    private String ramSize;

}

File: frontend/fontAngular/src/app/interfaces/server-details.ts
import { Status } from "./status.enum";

export class ServerDetails {
   id!:number;
   physicalServer!:string;
   serverName!:string;
   ipAddress!:string;
   userName!:string;
    password!:string;
    statusServer!:Status;
    ramSize!:string;
}
